docs/docs/usage/getting-started.md

```diff
@@ -10,7 +10,7 @@ title: Getting Started: Users
 This is an easy way to try Dotty without installing anything, directly in your browser.
 
 ### sbt
-The fastest way to create a new project compiled by Dotty is using [sbt (1.1.4+)](http://www.scala-sbt.org/)
+The fastest way to create a new project compiled by Dotty is using [sbt (1.3.6+)](http://www.scala-sbt.org/)
```

Comment:
The version number here is actually the minimum version needed for `sbt new` to work, but since it's so old and it can be confused with the version needed for dotty to work, we can just drop it I think:
```suggestion
The fastest way to create a new project compiled by Dotty is using [sbt](http://www.scala-sbt.org/)
```